legion/tests/e2e/robot/tests/training/resources/invalid/not_found_file.training.legion.yaml

```diff
@@ -0,0 +1,18 @@
+kind: ModelTraining
+id: test-downloading-training-data
+spec:
+  model:
+    name: test-training-data
+    version: 1.0
+  # Toolchain integration name from TEST_DATA_TI_ID variable in "legion/tests/stuff/training_stuff.sh" file
+  toolchain: training-data-helper
+  entrypoint: main
+  workDir: "."
+  hyperParameters:
+    input-file-location: "local/legion.project.yaml"
+    target-file-location: "legion.project.yaml"
+  data:
+    # Connection name from TEST_DATA_DIR_CONNECTION_ID variable in "legion/tests/stuff/training_stuff.sh" file
```

Comment:
TEST_DATA_FILE_CONNECTION_ID not DIR variable. Here and next occasions

Reply:
Thanks! Fixed